xapian-applications/omega/docs/.gitignore

```diff
@@ -9,4 +9,5 @@
 /quickstart.html
 /scriptindex.html
 /termprefixes.html
+/newformat.html
```

Comment:
This new entry should be added preserving the sorted order of the list (and similarly in `docs/Makefile.am` below).  I've pushed 1cc7fb53c519 to adjust these, so this is just FYI.